core/src/main/java/org/openstack4j/api/networking/ext/LbaasV2Service.java

```diff
@@ -0,0 +1,28 @@
+package org.openstack4j.api.networking.ext;
+
+/**
+ * LBaaS (Load Balancer as a Service) V2 support
+ * @author emjburns
+ */
+public interface LbaasV2Service {
```

Comment:
I wonder if all the sub-services need to have v2 in the method names - I would assume that LbaasV2Service would return V2 objects only.